### DISCUSSION TITLE
AspectOS V1.0.2 releases!

### DISCUSSION THREAD

**Discussion by AIGamesDeveloper** [AspectOSTeam/AspectOS#8] [AT Announcements]
The release of v1.0.2 of AspectOS is here, and it brings new things to the table

1. Bug fixes - There were a lot of bug fixes changed in this update. Everything should now be working again

2. Less lag - Yes, we have finally done it. AspectOS v1.0.2 provides less lag for users. It is still recommended to run this on a fast computer (something that is more powerful than an Intel Core i5 processor)

3. Removal of apps - we have gotten rid of the paint app and the wallpaper app due to them being a waste of our space.

4. Wallpapers - We have removed all of our wallpapers to reduce lag and we have a new Ukraine Flag Wallpaper on the project if you look right now #StandWithUkraine

We hope this update helps you! This is the update before the finale the end of the AspectOS v1.x updates. 
The next one will release next Tuesday with brand new bug fixes, more removal of apps, and a reduction of lag. Stay Tuned!

**Comment by udscminidisc** [comment]
Yay!
________________________________
From: AIGamesDeveloper ***@***.***>
Sent: Tuesday, March 15, 2022 7:12 PM
To: AspectOSTeam/AspectOS ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [AspectOSTeam/AspectOS] AspectOS V1.0.2 releases! (Discussion #8)


The release of v1.0.2 of AspectOS is here, and it brings new things to the table

  1.  Bug fixes - There were a lot of bug fixes changed in this update. Everything should now be working again

  2.  Less lag - Yes, we have finally done it. AspectOS v1.0.2 provides less lag for users. It is still recommended to run this on a fast computer (something that is more powerful than an Intel Core i5 processor)

  3.  Removal of apps - we have gotten rid of the paint app and the wallpaper app due to them being a waste of our space.

  4.  Wallpapers - We have removed all of our wallpapers to reduce lag and we have a new Ukraine Flag Wallpaper on the project if you look right now #StandWithUkraine

We hope this update helps you! This is the update before the finale the end of the AspectOS v1.x updates.
The next one will release next Tuesday with brand new bug fixes, more removal of apps, and a reduction of lag. Stay Tuned!

—
Reply to this email directly, view it on GitHub<https://github.com/AspectOSTeam/AspectOS/discussions/8>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AU7ZML3TDGYBOOABX6NFEMLVAEKOFANCNFSM5Q2GBNUQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
